public/app/features/plugins/extensions/utils.tsx

```diff
@@ -58,7 +60,12 @@ type ModalWrapperProps = {
   onDismiss: () => void;
 };
 
-export const wrapWithPluginContext = <T,>(pluginId: string, Component: React.ComponentType<T>, log: ExtensionsLog) => {
+export const wrapWithPluginContext = <T,>(
+  pluginId: string,
+  extensionTitle: string,
+  Component: React.ComponentType<T>,
+  log: ExtensionsLog
+) => {
```

Comment:
Personal preferences maybe but for 3 or more arguments we should use an object instead. WDYT?
```suggestion
export const wrapWithPluginContext = <T,>(
{
pluginId,
extensionTitle,
Component,
log
},
{
  pluginId: string;
  extensionTitle: string;
  Component: React.ComponentType<T>;
  log: ExtensionsLog;
}) => {
```

Reply:
Good point @hugohaggmark - totally agree, positional arguments start to get kinda hard to work with at this point. I'll change 👍 (Also for `renderWithPluginContext()` - https://github.com/grafana/grafana/pull/107515#discussion_r2181630788)

Reply:
Updated in https://github.com/grafana/grafana/pull/107515/commits/82cbd98d36e4c502c04605e8100495e72af32f56.